lua/nvim-tree.lua

```diff
@@ -469,6 +495,10 @@ local function validate_options(conf)
 end
 
 function M.setup(conf)
+  if vim.fn.has "nvim-0.7" == 1 then
```

Comment:
This test is backwards; 1 indicates success.

Reply:
Right, thank you! Changed